modules/scan/class-admin-bar-notice.php

```diff
@@ -182,7 +181,6 @@ public function has_threats() {
 			return null;
 		}
 
-		// Return true if there is at least one threat found.
```

Comment:
Not blocking, but why remove the comments? They seem like they would be helpful to read through when trying to learn the code.